docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -284,22 +258,17 @@ export default function StyledComponentsDeep() {
 Material-UI has a rich theme structure that you can leverage for
 the color manipulations, the transitions, the media queries, and more.
 
+By using the Material-UI's Provider, the theme will be available for the
+context as well (emotion or styled-components, depending on your configuraiton).
+
 We encourage to share the same theme object between Material-UI and your styles.
```

Comment:
I think that it would be great to cover the case where there is a conflict of theme structure between styled-components and Material-UI. I think that it's important for progressive adoption.

Reply:
You mean specifying that whatever `ThemeProvider` will be last, `styled-components`' or `Material-UI`'s that one will win?

Reply:
I was thinking of being very specific. "If you are already using a custom theme with styled-components or emotion, it might not be compatible with Material-UI. In these cases, you need to render the Material-UI's ThemeProvider **first**. This will ensure the theme structures are isolated. This is ideal for the progressive adoption of Material-UI's components"

Reply:
Got ya, adding it

Reply:
Done